integrationtest/mapper/orm-spring-uberjar/pom.xml

```diff
@@ -31,31 +31,24 @@
 
     <dependencyManagement>
         <dependencies>
-            <!-- Override the version of Hibernate ORM pulled by Spring Boot -->
-            <!--
-                Since we are importing a pom and not using it as a parent for this module,
-                we cannot use a version property like:
-                    <hibernate.version>${version.org.hibernate.orm}</hibernate.version>
-                or:
-                    <elasticsearch-client.version>${version.org.elasticsearch.client}</elasticsearch-client.version>
-                to override Spring's dependency version with the one we need. Instead, we are going import our bom to
-                override the versions. Placing it after the Spring bom - doesn't work and dependency versions
-                remain unchanged.
-            -->
-            <dependency>
-                <groupId>org.hibernate.orm</groupId>
-                <artifactId>hibernate-platform</artifactId>
-                <version>${version.org.hibernate.orm}</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
             <dependency>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-dependencies</artifactId>
                 <version>${version.org.springframework.boot}</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <!-- The Spring BOM uses a version of bytebuddy that's too old for Mockito to work correctly -->
```

Comment:
So we'll leave the ORM version to whatever spring has, and since this uberjar is not using mockito do we need to override the byte buddy?

Reply:
You're right on both counts, but I just copy-pasted from other Spring tests. If you want to fix the setup feel free to send another PR.